engine/src/main/scala/cromwell/engine/io/RetryableRequestSupport.scala

```diff
@@ -59,17 +59,17 @@ object RetryableRequestSupport {
   }
 
   def isGcs500(failure: Throwable): Boolean = {
-    val serverErrorPattern = ".*Could not read from gs.+500 Internal Server Error.*"
+    val serverErrorPattern = ".*500 Internal Server Error.*"
```

Comment:
TOL would a `String.contains` be a simpler/better way to express this intention?